src/NSFW.cpp

```diff
@@ -255,7 +255,9 @@ Napi::Value NSFW::Resume(const Napi::CallbackInfo &info) {
 void NSFW::pollForEvents() {
   while (mRunning) {
     uint32_t sleepDuration = 50;
-    if (!mPaused) {
+    if (mPaused) {
+      mQueue->clear();
```

Comment:
Can we stop allocating to mQueue when we're not reading from it?

Reply:
Should be able to do that, yeah. I'll look into that.